app/assets/javascripts/models/beta/story.js

```diff
@@ -64,7 +65,19 @@ export const getCompletedPoints = story => {
 
 export const isStoryNotEstimated = (storyType, estimate) => storyType === 'feature' && !estimate;
```

Comment:
Note for a future refactor (we don't need to do it now): let's just receive the `story` as the only parameter here.